.golangci.yml

```diff
@@ -0,0 +1,106 @@
+run:
+  skip-files:
+    - pkg/dns/dns.go
```

Comment:
Why are we skipping this file? We should ideally drop the exception, but at least add a comment.

Reply:
This file belongs to upstream, and eventually will not be part of this project.